### PR TITLE
Add target executable for `tesserato` editor tool

### PR DESCRIPTION
**Pull request by roby2014**
- changed `tools/` folder structure
- updated readme

### CODE REVIEW

File: tools/tesserato/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# tools/tesserato/CMakeLists.txt
+# Cubos tools (tesserato) build configuration
```
Comment (RiscadoA) [inline tools/tesserato/CMakeLists.txt:2]:
```suggestion
# Ttesserato build configuration
```

**Comment by rsubtil** [comment]
It's a good idea to bootstrap the base editor already, instead of a blank hello world. The current structure on the scene sample is a good start:
https://github.com/GameDevTecnico/cubos/blob/8656d1fee68b2ae1c89a1033eb0b10fb3de8e670/engine/samples/scene/main.cpp#L48-L62
But in this case only the two plugins should be added, since that's what we have so far.

**Comment by roby2014** [comment]
Sounds good to me.

File: tools/tesserato/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+set(TESSERATO_SOURCE
+    "src/main.cpp"
+)
+
+add_executable(tesserato ${TESSERATO_SOURCE})
+set_property(TARGET tesserato PROPERTY CXX_STANDARD 20)
+target_compile_features(tesserato PUBLIC cxx_std_20)
+target_link_libraries(tesserato PUBLIC cubos-core)
+target_link_libraries(tesserato PUBLIC cubos-engine)
```
Comment (RiscadoA) [inline tools/tesserato/CMakeLists.txt:12]: You can merge the two lines.
```suggestion
target_link_libraries(tesserato PUBLIC cubos-core cubos-engine)
```

**Review by RiscadoA — CHANGES_REQUESTED** [review]

File: tools/tesserato/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Tesserato build configuration
+
+set(TESSERATO_SOURCE
+    "src/main.cpp"
+)
+
+add_executable(tesserato ${TESSERATO_SOURCE})
+set_property(TARGET tesserato PROPERTY CXX_STANDARD 20)
+target_compile_features(tesserato PUBLIC cxx_std_20)
+target_link_libraries(tesserato PUBLIC cubos-core)
```
Comment (RiscadoA) [inline tools/tesserato/CMakeLists.txt:11]: You also need to remove this line (I couldn't remove it in the suggestion for some reason).

**Review by RiscadoA — APPROVED** [review]

**Pull request merged**